src/parser/jobs/rpr/modules/HarvestMoon.tsx

```diff
@@ -30,69 +30,69 @@ export class HarvestMoon extends Analyser {
 
-		// get the earliest unable to act window that falls within the provided inputWindow
+		// Find the earliest unable to act window that falls within the input window
 		const unableToActWindow = this.unableToAct.getWindows(inputWindow)[0]
 
-		// the window will be undefined if there are no unable to act windows left before the end of the inputWindow
+		// If there are no unable to act windows left before the end of the input window, this UTA will be undefined
+		// Check if our input window is long enough
 		if (unableToActWindow == null) {
-			// True if there are ADJUSTED_CAST_TIME milliseconds or more remaining in the inputWindow
 			return inputWindow.end - inputWindow.start >= ADJUSTED_CAST_TIME
 		}
 
-		// return true if there are ADJUSTED_CAST_TIME milliseconds between the beginning of the window being checked and the beginning of an unableToAct window
+		// Even if the input window was too short, check if there's a large enough time between the input window and UTA
 		if (unableToActWindow.start - inputWindow.start >= ADJUSTED_CAST_TIME) {
 			return true
 		}
 
-		// recurse the method, shrinking the window to the space between the end of the unable to act window and the end of the input window.
+		// Recurse the method, shrinking the window to the space between the end of the unable to act window and the end of the input window
 		return this.canChargeMoon({start: unableToActWindow.end, end: inputWindow.end})
 	}
 
-	private getExpectedUses(): number {
-		const ADJUSTED_CAST = this.data.actions.SOULSOW.castTime + SOULSOW_BUFFER
-		const invulnWindows = this.invulnerability.getWindows().filter((window) => window.end - window.start >=  ADJUSTED_CAST)
-
-		// will only run the window filter if there is a non-zero amount of unableToAct time during the fight.
-		if (this.unableToAct.getDuration({start: this.parser.pull.timestamp, end: this.parser.pull.timestamp + this.parser.pull.duration}) > 0) {
-			return invulnWindows.filter(window => this.canChargeMoon(window)).length + 1
-		}
-
-		return invulnWindows.length + 1
-	}
-
-	private getUsedPercentage(expectedUses: number): string {
-		return (this.harvestMoonCasts / expectedUses * 100).toFixed(2)
-	}
-
 	private onComplete() {
 		const expectedUses = this.getExpectedUses()
-		const hmUsesPercent = this.getUsedPercentage(expectedUses)
+		const harvestsUsedPercent = this.getUsedPercentage(expectedUses)
 
 		this.checklist.add(new Rule({
 			name: <Trans id="rpr.harvestmoon.checklist.title">
-				Use <DataLink action="SOULSOW" /> during downtime.
+				Use <DataLink action="SOULSOW" /> during downtime
 			</Trans>,
 			description: <Trans id="rpr.harvestmoon.checklist.description">
-				Harvest Moon is one of your highest potency abilities. Aim to get at least one use each time the boss is targetable.
+				<DataLink action="HARVEST_MOON"/> is one of your highest damage abilities. Aim to get at least one use each time the boss is targetable.
```

Comment:
I think this suggestion is a little weird, I took a stab at it but I think it still needs more work:
`<DataLink action="HARVEST_MOON"/> is one of your highest damage abilities. Use it before the end of the fight or before any downtime that is long enough to recharge it.`

Also, this module I think doesn't care if someone just charges soul sow during uptime, and maybe it should? It's incorrect to charge it during uptime.